politeiad/cmd/politeiatlog/filesystem.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"github.com/decred/politeia/politeiad/backend/tlogbe/store"
+)
+
+// This tool is assuming that the store is a fileSystem for now. Politeiatlog
+// will have cert flags later on for connecting to a SQL database.
+//
+// fileSystem implements the Blob interface.
+type fileSystem struct {
```

Comment:
Why are you re-implementing this?

Reply:
I thought the way to go here was to implement the blob interface locally for the `politeiatlog` tool. Filesystem for now, and later a mysql implementation. Reusing it from `politeiad/backend/tlogbe/store/filesystem/` didn't seem too correct as it would couple the code together. Am I mistaken here? should we reuse it?